src/classes/window/fullscreen.vala

```diff
@@ -50,13 +50,20 @@ namespace pdfpc.Window {
         protected bool faded_to_black = false;
 
         /**
-         * Fixed layout
+         * Overlay layout. Holds all drawing layers (like the pdf,
+         * the pointer mode etc)
          */
-        protected Gtk.Fixed fixed_layout;
+        protected Gtk.Overlay overlay_layout;
 
-        public void add_to_fixed(Gtk.Widget w, int x, int y) {
-            this.fixed_layout.put(w, x, y);
-        }
+        /**
+         * Drawing area for pointer mode
+         */
+        public Gtk.DrawingArea drawing_surface { get; protected set; }
+
+        /**
+         * Video area for playback
+         */
+        public View.Video video_surface { get; protected set; }
```

Comment:
Right. I didn't mean that we should emphasize that it's a `Gtk.Fixed`, but that this isn't an object meant for displaying *one* video but rather one for displaying *all*. (Actually, while reviewing, I suspected the first and checked whether a file with two videos on one slide still works before bothering to figure out what this actually is..)